apps/website/src/pages/docs/textarea.mdx

```diff
@@ -24,6 +21,36 @@ Text areas (or multi-line fields) are larger text fields that can accommodate mo
 
 It used to be possible to freely resize the text area horizontally in addition to vertically; this pattern is now deprecated, and text areas only allow vertical resizing for better predictability. The resize handle is optional; it may be omitted if resizing of the text area is not desired.
 
+<LiveExample src='Textarea.scroll.tsx'>
+  <AllExamples.TextareaScrollExample client:load />
+</LiveExample>
+
+If the area is shrunk to a size smaller than the amount of text within, a scrollbar appears to keep navigation possible. This behavior is especially important for text areas without a resizing handle, to avoid making text editing tedious.
+
+## Variants
+
+### Status
+
+<LiveExample src='Textarea.status.tsx'>
+  <AllExamples.TextareaStatusExample client:load />
+</LiveExample>
+
+If status messages are required, the styling is once again similar to [inputs](input). For the default textarea, the bottom border gains the color of the alert level, as well as the corresponding icon and message.
+
+### Inline
+
+<LiveExample src='Textarea.inline.tsx'>
+  <AllExamples.TextareaInlineExample client:load />
+</LiveExample>
+
+It’s possible to use textareas inline, in forms for example. he status displays a bit differently. For these textareas the status border remains the same, but the icon displays horizontally centered inside the area, at the far right.
```

Comment:
Is this level of depth needed for our documentation site, the describing how it looks? Because users no longer have to make their stuff look how we want; it already does. But if you leave it, there's a "T" missing at the beginning of the second sentence.

But it could be something like "It's possible to use text areas inline. This could be useful in styling forms."